Util/RNG.gd

```diff
@@ -149,3 +149,19 @@ static func randomFemaleName():
 static func pickHashed(ar, thehash:int):
 	thehash = thehash % ar.size()
 	return ar[thehash]
+
+static func shouldCondomBreakWhenCumInside(characterPenetrated, chance: float = 20.0) -> bool:
```

Comment:
Move this function into the baseCharacter class. RNG.gd is just a utlity class that is not supposed to know -anything- about the game. Basically you should be able to grab this file and add it into a new project and it should just work. Right now it won't